src/frontend/src/test-e2e/views.ts

```diff
@@ -221,7 +221,7 @@ export class DeviceSettingsView extends View {
   async removeDisabled(): Promise<void> {
     await this.browser
       .$("button[data-action='remove']")
-      .waitForEnabled({ reverse: true });
+      .waitForDisplayed({ reverse: true });
```

Comment:
@frederikrothenberger the function suggests it's checking that the remove button is disabled; can you explain why it's now checking whether the button is not displayed?

Reply:
Ah, I think this was a late UI change. The remove button is now no longer displayed (and there is an explanatory text now why the device cannot be removed). The function should be renamed.

Reply:
Ok thanks, can you update the code?

Reply:
See #749.